components/map/MapZones.tsx

```diff
@@ -13,26 +13,36 @@ type Props = {
 }
 
 const MapZoneShapes = ({ udrData }: Props) => {
+  const shownUdrData = useMemo(
+    () => ({
+      ...udrData,
+      features: udrData.features.filter(
+        (udr) => udr.properties?.status !== MapZoneStatusEnum.planned,
```

Comment:
Wouldn't it be easier to filter here directly only `active` zones and use that below?
I suspect that the zone can not be `active` and `planned` at the same time. Also, correct me if I'm wrong, but we probably also don't want to show **in**active zones with UDR `0`